nemo/collections/asr/models/msdd_models.py

```diff
@@ -190,34 +211,56 @@ def __setup_dataloader_from_config(self, config):
             sample_rate=config['sample_rate'], int_values=config.get('int_values', False), augmentor=None
         )
 
-        if 'manifest_filepath' in config and config['manifest_filepath'] is None:
+        if (
+            'manifest_filepath' in config
+            and config['manifest_filepath'] is None
+            and (
+                'synthetic' not in config
+                or config['synthetic'] == False
+                or self.cfg_msdd_model.train_ds.include_base_ds
+            )
+        ):
             logging.warning(f"Could not load dataset as `manifest_filepath` was None. Provided config : {config}")
             return None
```

Comment:
If there is no dataset, shouldn't we raise error here?

Reply:
I can change this to throw an error (currently if the manifest file isn't included it will return None here and then there will be a file not found error)